examples/README.md

```diff
@@ -45,7 +45,7 @@ OPTIONAL - Rebuilding the generated code
 
 1 First [install protoc](https://github.com/google/protobuf/blob/master/README.md)
   - For now, this needs to be installed from source
-  - This is will change once proto3 is officially released
+  - This will change once proto3 is officially released
```

Comment:
This is actually not true anymore...

Can you remove these two lines and update the link for `install protoc`?
```markdown
First [install protoc](https://github.com/google/protobuf/blob/master/README.md#protocol-compiler-installation)
```

Reply:
It's my pleasure.